src/main/java/org/snomed/snowstorm/core/data/services/traceability/TraceabilityConsumer.java

```diff
@@ -47,7 +47,7 @@ void sendInBatches(Activity activity) {
 			logger.info("Sending batch with start index {}.", idx);
 			List<Activity.ConceptActivity> changes = activity.getChanges().subList(idx, Math.min(idx + maxConceptActiviesPerMessage, changeListSize));
 			Activity activitySlice = new Activity(activity.getUserId(), activity.getBranchPath(),
-			activity.getCommitTimestamp(), activity.getSourceBranch(), activity.getActivityType());
+			activity.getCommitTimestamp(), activity.getSourceBranch(), activity.getActivityType(), false);
```

Comment:
@markrynbeek Shouldn't we send an activity with changeListComplete set to true in the last message?

Reply:
Please ignore this one.